### ISSUE TITLE
Inconsistent EventIDs between multiple demos

### ISSUE THREAD

**Issue by marcouderzo** [marcouderzo/CSDemoParser#3]
Some eventIDs do not match between multiple demos

**Demo 1**

```
descriptors {
  eventid: 132
  name: "grenade_thrown"
  keys {
    type: 4
    name: "userid"
  }
  keys {
    type: 1
    name: "weapon"
  }

descriptors {
  eventid: 134
  name: "weapon_reload"
  keys {
    type: 4
    name: "userid"
  }
```

**Demo 2**

```
weapon_reload
{
 eventid: 132
 userid: AdreN (id:12)
  position: -1246.599487, 2599.400879, 91.277603
  facing: pitch:22.439575, yaw:188.459473
  team: CT
}
```

**Comment by marcouderzo** [comment]
Using events name instead of IDs for now.

**Comment by marcouderzo** [comment]
close #3, no need to use msg IDs as pDescriptor->name() can be used instead

**Comment by marcouderzo** [comment]
close #3. Confirmed, we used a diffchecker and seen a ton of inconsistencies between demos, as far as event IDs are concerned. Also in some demos some events are not present, in others there are events never documented before. We cannot do anything about it, as *the descriptors are embedded in the demo itself* .

For more information, check out an example diff [here](https://www.diffchecker.com/zC7OIIba#left-2).
